src/lavinmq/shovel/shovel.cr

```diff
@@ -162,6 +165,36 @@ module LavinMQ
       abstract def started? : Bool
     end
 
+    class DestinationWrapper < Destination
```

Comment:
`DestinationWrapper` sounds like an abstract name to me. Maybe we can figure out something more descriptive? `MultiDestination` or something.. hm.

Reply:
`MultiDestination`, or `UniversalDestination` could work

Reply:
chose MultiDestinationHandler